### PR TITLE
#CX-10595 : Incoming Call Details: Color Contrast: Blue focus indicator on the list items fails contrast ratio

### PR DESCRIPTION
**Pull request by arunmohanpk**
CX-10595
## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: web-components/src/components/combobox/ComboBox.ts [hunk 1]
```diff
@@ -882,6 +884,16 @@ export namespace ComboBox {
       }
     }
 
+    private setSelectedatribute(option: string | OptionMember | undefined) {
```
Comment (parsoman) [inline web-components/src/components/combobox/ComboBox.ts:887]: Fix the spelling of setSelectedAttribute

File: web-components/src/components/combobox/ComboBox.ts [hunk 2]
```diff
@@ -882,6 +884,20 @@ export namespace ComboBox {
...
+      }
+      this.lists?.forEach((list, index) => {
+        if (list?.id === optionId) {
+          list?.setAttribute("selected", "true");
+        } else {
+          list?.setAttribute("selected", "false");
+        }
+      });
+    }
+
```
Comment (vinitkumargoel) [inline web-components/src/components/combobox/ComboBox.ts:900]: Please add unit tests

File: web-components/src/components/combobox/tokens/md-combobox-tokens.js [hunk 3]
```diff
@@ -58,17 +58,25 @@ const combobox = {
...
+        light: colors.gray[100].name,
+        dark: colors.gray[90].name
+      },
+      "selected-bg-color": {
+        light: colors.blue[50].name,
+        dark: colors.blue[40].name
+      },
       "checked-bg-color": {
         light: colors.theme[50].name,
         dark: colors.theme[60].name
```
Comment (vinitkumargoel) [inline web-components/src/components/combobox/tokens/md-combobox-tokens.js:82]: If you are changing lm token change same in md token

**Review by vinitkumargoel — CHANGES_REQUESTED** [review]

File: web-components/src/components/combobox/tokens/md-combobox-tokens.js (hunk 3 above)
Comment (arunmohanpk) [reply web-components/src/components/combobox/tokens/md-combobox-tokens.js:82]: reverted the change to original
checked-bg-color

**Review by vinitkumargoel — APPROVED** [review]

**Pull request merged**